client/signup/steps/plans/index.jsx

```diff
@@ -138,6 +181,17 @@ module.exports = React.createClass( {
 	},
 
 	render: function() {
+		const showPlanFeatures = config.isEnabled( 'manage/plan-features' );
+
+		if ( showPlanFeatures ) {
+			return (
+				/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
@lamosty maybe add a modifier class, if you need something to differentiate this from the original plans step